SPECS/moby-containerd/moby-containerd.spec

```diff
@@ -86,6 +85,9 @@ fi
 %config(noreplace) %{_sysconfdir}/containerd/config.toml
 
 %changelog
+* Tue Feb 28 2023 Bala <balakumaran.kannan@microsoft.com> - 1.6.19-1
```

Comment:
Can we please modify the PR title to something like:
`Upgrade moby-containerd to v1.6.19 to address CVE-2023-25173`